src/SFA.DAS.Apprenticeships.Web/Models/SignedOutViewModel.cs

```diff
@@ -0,0 +1,14 @@
+namespace SFA.DAS.Apprenticeships.Web.Models;
+
+public class SignedOutViewModel
```

Comment:
On provider signout we redirect to the main provider app, should this not follow the same behaviour?